ComfyUI/utils/lora_merger.py

```
@@ -0,0 +1,187 @@
+"""
+LoRA Merger Utility for ComfyUI
+
+This module provides LoRA merging functionality using ComfyUI's existing LoRA infrastructure.
+It can be used as a standalone utility or imported by other tools.
+"""
+
+import os
+import sys
+import logging
+import torch
+import tempfile
```

Ruff (0.11.9)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)